tests/build.gradle

```diff
@@ -16,6 +16,7 @@ tasks.withType(Test) {
 }
 
 task testLean(type: Test) {
+    exclude '**/*Java*'
```

Comment:
Ah, yes I already investigated that. 
I already do it with provider repos kafka, cloudant and alarms.

For this repo, even if we re-arrange things will be a lot of changes as we need to build and run gradle, too complex in my opinion.

Not that simple, @rabbah already was down this path for the trivial ones the runtimes, and we hit some problems.

I think this is the best option right now, to just push from Travis it only takes 4 minutes, also once I setup jenkins in ASF we could see moving it there and take our 4 minutes back.